extensions/connect-to-vpn/src/index.tsx

```diff
@@ -105,6 +105,12 @@ export default function Command() {
 
         Promise.all(promises).then((services) => {
           const networkServices = services.reduce((acc, service) => {
+            if (getPreferenceValues().hideDisabled && service.status == "invalid") {
+              // Remove notice about disabled services
+              setHeader(undefined);
+              return acc;
+            }
+
```

Comment:
Thanks for the response at https://github.com/raycast/extensions/pull/7004#issuecomment-1588945552. If I were in your shoes I would not place the logic in the middle of reduce wrapped in promises as it'd make it harder to make sense. I would rather tweak only presentational logic in render. See the suggestion that I post aside.